express/features/client-details-update.feature

```diff
@@ -0,0 +1,149 @@
+Feature: Client details update pages
+
+  Background:
+    Given that the user is on the "/sign-in" page
+    And the user submits the email "registered@gds.gov.uk"
+    And the user submits the password "this-is-not-a-common-password"
+    And the user submits the security code "123456"
+    Then they should be redirected to a page with the title "Client details - GOV.UK One Login"
+
+  Scenario: The user wants to change their service name
+    When they click on the link with the URL starting with "/change-service-name"
+    Then they should be redirected to a page with the title "Change service name - GOV.UK One Login"
+
+  Scenario: The user wants to change their re-direct URIs
+    When they click on the link with the URL starting with "/change-redirect-uris"
+    Then they should be redirected to a page with the title "Change redirect URIs - GOV.UK One Login"
+
+  Scenario: The user wants to change their user attributes
+    When they click on the link with the URL starting with "/change-user-attributes"
+    Then they should be redirected to a page with the title "Change user attributes - GOV.UK One Login"
+
+  Scenario: The user wants to change their public key
+    When they click on the link with the URL starting with "/change-public-key"
+    Then they should be redirected to a page with the title "Change public key - GOV.UK One Login"
+
+  Scenario: The user wants to change their Post logout redirect URIs
+    When they click on the link with the URL starting with "/change-post-logout-uris"
+    Then they should be redirected to a page with the title "Change post logout redirect URIs - GOV.UK One Login"
+
+  Scenario: The user wants to see more details about the user attribute
+    When they click on the link with the URL starting with "/change-user-attributes"
+    Then they should be redirected to a page with the title "Change user attributes - GOV.UK One Login"
+    When they click on the "user attributes (opens in new tab)" link that opens in a new tab
+    Then they should be directed to the URL "https://docs.sign-in.service.gov.uk/integrate-with-integration-environment/choose-which-user-attributes-your-service-can-request/"
+    And they should see the text "Choose which user attributes your service can request"
+
+  Scenario: The user doesn’t want to save their user attributes
+    When they click on the link with the URL starting with "/change-user-attributes"
+    Then they should be redirected to a page with the title "Change user attributes - GOV.UK One Login"
+    When they click on the "Cancel" link
+    Then they should be redirected to a page with the title "Client details - GOV.UK One Login"
```

Comment:
This scenario doesn't really test anything as the email checkbox is checked by default.
It should be modified to check that a change is registered, for instance
```
Given the "phone" user attribute is not selected
The user clicks the link....
The user checks the "Phone number" checkbox
click the Confirm button
Then the "phone" user attribute should be selected
```

And another test to check that the email attribute is de-selected